content/en/synthetics/private_locations.md

```diff
@@ -23,40 +23,99 @@ This feature is in public beta for API Tests. The equivalent feature for Browser
 
 Private locations allow you to monitor internal-facing applications or any private URLs that aren’t accessible from the public internet. They can also be used to create a new custom Synthetics location.
 
-## Setup
+By default, every second, your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test depending on the frequency defined in the configuration of the test, and returns the test results to Datadog’s servers.
 
-The private location worker is shipped as a Docker container, so it can run on a Linux based OS or Windows OS if the Docker engine is available on your host and can run in Linux containers mode.
+Once you created a private location, configuring a [Synthetics test][1] to run from a private location is completely identical to how you would assign tests to Datadog managed locations.
 
-By default, every second, your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test depending on the frequency defined in the configuration of the test, and returns the test results to Datadog’s servers.
+## Prerequisites
+
+### Datadog Private Locations Endpoints
```

Comment:
I think it's worth updating this section to take advantage of the new US/EU functionality like https://docs.datadoghq.com/getting_started/logs/
(so if you switch between the selectors, it shows you the content you select)

It works like tabs except this:
```
{{< site-region region="us" >}}
US site content and options
{{< /site-region >}}
{{< site-region region="eu" >}}
EU site content and options
{{< /site-region >}}
```